tools/README.md

```diff
@@ -1,12 +1,42 @@
-### Contents of `tools/`
+# Contents 
 
-This directory contains various tools for use by both developers and users.
-A description of each of the files in this directory is given in the table below.
+This `tools` directory contains various tools for use by both developers and users.
+
+Files located at the root of `tools`:
+- `configure_dependencies.py`: Script for installing OpenSn dependencies
+- [lua-input-style.md](./lua-input-style.md): Notes on how to format `lua` input files. 
+
+### Tools for Developer
+
+Files are located in `tools/developer`.
+
+A description of the files or sub-directory is given in the table below.
+
+| File or Directory  | Description                                             |
+|--------------------|---------------------------------------------------------|
+| `lsan.supp`        | Leak sanitizer suppression file (used in git workflows) |
+| `ignorelist.txt`   | (used in CMakepresets.json)                             |
```

Comment:
List of source-level entities that should be ignored by sanitizers.

Reply:
added text